python/ray/air/checkpoint.py

```diff
@@ -821,6 +834,11 @@ def get_preprocessor(self) -> Optional["Preprocessor"]:
 
```

Comment:
Don't we need to return `self._override_preprocessor` in `get_preprocessor`?

Reply:
Not needed since `to_dict` and `to_directory` already set the preprocessor.